beta/nncf/tensorflow/sparsity/schedulers.py

```diff
@@ -152,6 +152,37 @@ def _init_exp(initial_sparsity, max_sparsity, sparsity_steps=20):
         return a, k
 
 
+@SPARSITY_SCHEDULERS.register("adaptive")
+class AdaptiveSparsityScheduler(SparsityScheduler):
```

Comment:
@andrey-churkin, could you review `AdaptiveSparsityScheduler`.

Reply:
Removed it after pythorch / tf unification